src/encoded/static/components/static-pages/components/StatisticsPageViewBody.js

```diff
@@ -0,0 +1,1092 @@
+            this.setState({
+                'externalTermMap' : _.object(_.map(resp['@graph'] || [], function(award){
+                    return [ award.center_title, award.project !== '4DN' ];
+                }))
+            });
+        });
+    }
+
+    render(){ return <StatsViewController {...this.props} {...this.state} />; }
+
+}
+
+
+
+class UsageChartsCountByDropdown extends React.PureComponent {
+
+    constructor(props){
+        super(props);
+        this.handleSelection = this.handleSelection.bind(this);
+    }
+
+    handleSelection(evtKey, evt){
+        const { changeCountByForChart, chartID } = this.props;
+        changeCountByForChart(chartID, evtKey);
+    }
+
+    render(){
+        const { countBy, chartID } = this.props;
+        const currCountBy = countBy[chartID];
+
+        const menuOptions = new Map();
+
+        if (chartID === 'experiment_set_views' || chartID === 'file_views'){
+            menuOptions.set('views',        <React.Fragment><i className="icon fas icon-fw icon-eye mr-1"/>Detail View</React.Fragment>);
+            menuOptions.set('list_views',   <React.Fragment><i className="icon fas icon-fw icon-list mr-1"/>Appearance within first 25 Search Results</React.Fragment>);
+            menuOptions.set('clicks',       <React.Fragment><i className="icon far icon-fw icon-hand-point-up mr-1"/>Search Result Click</React.Fragment>);
+        } else if (chartID === 'file_downloads'){
+            menuOptions.set('experiment_type', <React.Fragment><i className="icon far icon-fw icon-folder mr-1"/>Experiment Type</React.Fragment>);
+            menuOptions.set('geo_country',     <React.Fragment><i className="icon fas icon-fw icon-globe mr-1"/>Country</React.Fragment>);
+            menuOptions.set('filetype',     <React.Fragment><i className="icon far icon-fw icon-file-alt mr-1"/>File Type</React.Fragment>);
+        } else {
+            menuOptions.set('views',    <React.Fragment><i className="icon icon-fw fas icon-eye mr-1"/>View</React.Fragment>);
+            menuOptions.set('sessions', <React.Fragment><i className="icon icon-fw fas icon-user mr-1"/>User Session</React.Fragment>);
+        }
+
+        const dropdownTitle = menuOptions.get(currCountBy);
+
+        return (
+            <div className="inline-block mr-05">
+                <DropdownButton data-tip="Count By" size="sm" id={"select_count_for_" + chartID}
+                    onSelect={this.handleSelection} title={dropdownTitle}>
+                    {_.map([ ...menuOptions.entries() ], function([ k, title ]){
+                        return <DropdownItem eventKey={k} key={k}>{ title }</DropdownItem>;
+                    })}
+                </DropdownButton>
+            </div>
+        );
+    }
+}
+
+
+export const UsageStatsView = React.memo(function UsageStatsView(props){
+    const {
+        loadingStatus, mounted, session, groupByOptions, handleGroupByChange, currentGroupBy, respTrackingItem, windowWidth,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['respTrackingItem' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5603033)